packages/core/src/service/services/product-option-group.service.ts

```diff
@@ -51,10 +51,14 @@ export class ProductOptionGroupService {
     ): Promise<Array<Translated<ProductOptionGroup>>> {
         const findOptions: FindManyOptions = {
             relations: relations ?? ['options'],
+            where: {
+                deletedAt: IsNull(),
+            },
         };
         if (filterTerm) {
             findOptions.where = {
                 code: Like(`%${filterTerm}%`),
+                deletedAt: IsNull(),
```

Comment:
suggestion: rather than repeat the clause, it would be marginally more maintainable to spread the existing where clause:

```ts
findOptions.where = {
  code: Like(`%${filterTerm}%`),
  ...findOptions.where,
}
```
which means if we every add any other where clauses up on line 54, we don't need to duplicate them on line 61.